src/pageServices/connector.ts

```diff
@@ -50,7 +50,7 @@ class Connector {
         .then((response) => response)
         .catch((error: Error) => {
           if (error.message === SCRIPT_ERROR.NO_RESPONSE && isUndefined(onResponse)) return null;
-          if (error.message === SCRIPT_ERROR.NO_CONNECTION && action !== "PING_SCRIPT" && action !== "CHECK_SESSION") {
+          if (error.message === SCRIPT_ERROR.NO_CONNECTION && action !== "PING_SCRIPT" && action !== "CHECK_SESSION" && action !== "KILL_HIGHLIGHT") {
```

Comment:
I’d suggest to create constants/enums for all these strings, because as I see we have them in some other places of our application as well

Reply:
Agree with you, but I'd like to do this refactoring in a new PR